__tests__/ratingRoutes.test.js

```diff
@@ -57,4 +83,101 @@ describe('Check ratings', () => {
         done();
       });
   });
+
+  describe('Create Rating', () => {
+    it('should throw a 500 when an error occurs on the server', (done) => {
+      const stub = sinon.stub(Rating, 'create').rejects(new Error('Foreign Key constraint'));
+      chai
+        .request(app)
+        .post(rateUrl)
+        .set('Authorization', `Bearer ${verifiedUserToken}`)
+        .send(mockRates[0])
+        .end((err, res) => {
+          expect(res.status).to.eql(500);
+          stub.restore();
+          done();
+        });
+    });
+
+    it('should create a rating successfully', (done) => {
+      chai
+        .request(app)
+        .post(`${rateUrl}`)
+        .set('Authorization', `Bearer ${verifiedUserToken}`)
+        .send(mockRates[0])
+        .end((err, res) => {
+          expect(res.status).to.equal(201);
+          const { article } = res.body;
+          expect(article.message).to.be.eql('Your rating has been added successfully');
+          expect(article).to.have.property('articleId');
+          expect(article).to.have.property('rate');
+          expect(article).to.have.property('firstName');
+          expect(article).to.have.property('lastName');
+          done();
+        });
```

Comment:
I've given this feedback about writing specific assertions for our tests. Please fix

Reply:
Okay I would. Thanks

Reply:
I'm sure `seededUsers[0]` has a first and last name which is the specific value returned in the response.

How about the rating you submitted, does that not have a specific value?

Reply:
seederUsers[0] is just the login details of the user. It could be any user. 

I have fixed the assertion for the rating value I submitted